README.md

```diff
@@ -42,6 +42,15 @@ This section describes the workflow we are going to follow when working in a new
 
 Forking the project will create a copy of that project in your own GitHub account, you will commit your work against your own repository.
 
+#### Quick start
+
+1. Navigate to the project folder.
+2. Run `npm install`
```

Comment:
We're using `yarn`. Let's stick with it to avoid conflicts.

Reply:
I'm looking at it as "I'm new, reading through top to bottom, what do I need to get started now". And the the available scripts is more like a glossary/reference after first success.